source/MaterialXRuntime/RtNode.cpp

```diff
@@ -17,6 +17,28 @@ namespace MaterialX
 namespace
 {
     static const RtToken NODEDEF("nodedef");
+
+    static const RtToken XPOS_METADATA("xpos");
```

Comment:
These metadatas are redefined in both RtNode and RtNodeGraph. Should I store them in some common location? If so, I wasn't sure where that should be.